### PR TITLE
Update session recording settings

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

From user feedback and some session recordings ([#1](https://app.posthog.com/sessions/play?id=1760bd675e53be-0e1242623a085a-16144b58-13c680-1760bd675e6bdc)),
- Adds a few clarifications around requirements for session recordings.
- Simplifies the process for giving feedback on the session recording feature (seems like opening a GH is too much friction), using the command palette.
- Migrates project settings page to TS.

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

**Review by macobo — APPROVED** [review]

**Comment by macobo** [comment]
Did you forget to push the feedback sharing change?

File: frontend/src/scenes/project/Settings/index.tsx [hunk 1]
```diff
@@ -16,18 +16,51 @@ import { CodeSnippet } from 'scenes/ingestion/frameworks/CodeSnippet'
...
     const { currentTeam } = useValues(teamLogic)
     const { resetToken } = useActions(teamLogic)
     const { location } = useValues(router)
+    const { user } = useValues(userLogic)
+
+    const { activateFlow, showPalette } = useActions(commandPaletteLogic)
+
+    const shareFeedback = (instruction: string = "What's on your mind?"): void => {
+        showPalette()
+        activateFlow({
```
Comment (macobo) [inline frontend/src/scenes/project/Settings/index.tsx:36]: Aha - didn't see this when originally reviewing.

This shouldn't live in this component, but as an action/listener under commandPaletteLogic.

Comment (paolodamico) [reply frontend/src/scenes/project/Settings/index.tsx:36]: Addressed on #2639

Comment (Twixes) [reply frontend/src/scenes/project/Settings/index.tsx:36]: TBH I'd prefer this to live inside the component. Overall with the current palette logic approach everything goes into `commandPaletteLogic`, which gets more and more bloated with every release.

**Pull request merged**